website/static/data/event-poster-card.json

```diff
@@ -1,6 +1,6 @@
 {
-  "show": false,
-  "expire": "2021-10-25",
-  "image": "https://static.apiseven.com/202108/meetup-card.png",
-  "link": "/blog/2021/10/18/meetup"
+  "show": true,
+  "expire": "2021-12-27",
+  "image": "https://static.apiseven.com/202108/1639621424088-075860f6-6e80-48de-a18d-aa70fdf2dc4c.png",
+  "link": "/blog/2021/12/16/apisix-with-rocketmq-meetup"
```

Comment:
Just a reminder from me that:
Do we need a maintenance document explaining the steps to add pop-ups?

Reply:
Sure~ It would be great to have them explained in `https://github.com/apache/apisix-website/blob/master/website/docs/general/blog-contributing-guide.md`